reaktive/src/darwinCommonMain/kotlin/com/badoo/reaktive/scheduler/MainScheduler.kt

```diff
@@ -59,7 +59,7 @@ internal class MainScheduler : Scheduler {
         }
 
         override fun cancel() {
-            operations.clear()
+            operations.clear(dispose = true)
```

Comment:
`true` is a default value.

Reply:
Yep, as I mentioned in the description, there is a bug and we must supply the value in shared source sets.

Reply:
Same for atomics